### DISCUSSION TITLE
How to improve server FPS/Performance?

### DISCUSSION THREAD

**Discussion by johanes** [thijsvanloef/palworld-server-docker#505] [Q&A] [answered]
Hi everyone I have 2 questions,

1- I would like to know if there is a way to improve the RTT and Jitter ms from the server with configs.
2- Where in the palworld-server-docker can I set the property `NetServerMaxTickRate=120`?

I found this on rededit regarding the server FPS but could not find the location in the docker container nor the game files in the server:

> Steam\steamapps\common\PalServer\Pal\Saved\Config\LinuxServer
>Once in the LinuxServer Folder, type: "nano Engine.ini "
>When the file opens, Copy and paste this code at the end.
>[/Script/OnlineSubsystemUtils.IpNetDriver]
>`NetServerMaxTickRate=120`

![image](https://github.com/thijsvanloef/palworld-server-docker/assets/3415718/6ec0680e-3628-4c0a-bf88-a05613a2b068)

**Comment by thijsvanloef** [comment, marked as answer]
Hi @johanes, you are probably looking to change the palworld Engine settings, You can find the documentation here: https://palworld-server-docker.loef.dev/getting-started/configuration/engine-settings

**Reply by johanes** [reply]
Thanks for the answer, and also thanks for making this docker image so easy to manage!

**Reply by thijsvanloef** [reply]
Thank you for the kind words, you are very welcome :)
